src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,139 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import EastIcon from "@mui/icons-material/East";
+import SwipeableViews from "react-swipeable-views-react-18-fix";
+import { autoPlay } from "react-swipeable-views-utils";
+
+const AutoPlaySwipeableViews = autoPlay(SwipeableViews);
+
+const images = [
+  {
+    label: "Carry your Funk1",
+    imgPath:
+      "https://images.unsplash.com/photo-1537944434965-cf4679d1a598?auto=format&fit=crop&w=400&h=250&q=60",
+  },
+  {
+    label: "Carry your Funk2",
+    imgPath:
+      "https://images.unsplash.com/photo-1512341689857-198e7e2f3ca8?auto=format&fit=crop&w=400&h=250&q=60",
+  },
+  {
+    label: "Carry your Funk3",
+    imgPath:
+      "https://images.unsplash.com/photo-1537996194471-e657df975ab4?auto=format&fit=crop&w=400&h=250",
+  },
+  {
+    label: "Carry your Funk4",
+    imgPath:
+      "https://images.unsplash.com/photo-1512341689857-198e7e2f3ca8?auto=format&fit=crop&w=400&h=250&q=60",
+  },
+];
+
+function Carousel() {
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const handleStepChange = (step) => {
+    setActiveStep(step);
+  };
+
+  return (
+    <Box
+      sx={{
+        position: "relative",
+        margin: "1%",
+        overflow: "hidden",
+        borderRadius: "25px",
+      }}
+    >
+      <AutoPlaySwipeableViews
+        index={activeStep}
+        onChangeIndex={handleStepChange}
+        enableMouseEvents
+      >
+        {images.map((step, index) => (
+          <div key={step.label}>
+            {Math.abs(activeStep - index) <= 2 ? (
```

Comment:
Should be abstracted to a variable

Reply:
I have abstracted the styles into a variable. Is that what you meant?

Reply:
No the equation that starts with math abs what does it mean ? Define it by a variable, someone reading your code doesn't have to analyse what that means they can just look at the var name and know